### PR TITLE
fix: remove remaining sygroup references

### PR DESCRIPTION
**Pull request by Yelinz**

### CODE REVIEW

File: frontend/app/dashboard/template.hbs [hunk 1]
```diff
@@ -95,7 +95,7 @@
         {{/card.header}}
         {{#card.body}}
           <p>
-            Adfinis SyGroup AG<br>
```
Comment (velrest) [inline frontend/app/dashboard/template.hbs:98]: Is the AG also removed from the official name?

Comment (Yelinz) [reply frontend/app/dashboard/template.hbs:98]: Dont know but the current name also doesn't have AG in it

Comment (velrest) [reply frontend/app/dashboard/template.hbs:98]: So do we leave it as is or change it to AG?

Comment (Yelinz) [reply frontend/app/dashboard/template.hbs:98]: Ill ask

Comment (Yelinz) [reply frontend/app/dashboard/template.hbs:98]: Changed it to AG

**Pull request merged**